docs/use-mainnet/fund.mdx

```diff
@@ -94,4 +105,60 @@ Not all available tokens will drip directly onto Linea. If they are, you will se
 
 :::
 
-Specific tokens require specific bridges. If you want to bridge from Goerli to Linea, you can find the tokens, contract addresses, and associated bridges [here](./info-contracts.mdx#token-contract-addresses-and-bridges).
\ No newline at end of file
+Specific tokens require specific bridges. If you want to bridge from Goerli to Linea, you can find the tokens, contract addresses, and associated bridges [here](./info-contracts.mdx#token-contract-addresses-and-bridges).
+  </TabItem>
+  
+  <TabItem value="mainnet">
+  Buying tokens directly on Linea is a straightforward process, using MetaMask Portfolio's Buy feature. 
+  
+  :::info
+  ### If you already have tokens on another network, you can [follow the instructions here](./bridges-of-linea/bridging-non-dev.mdx) to bridge them to Linea.
+  :::
+
+**Linea purchases in MetaMask Portfolio's Buy feature are available worldwide.**
+
+<div class="center-container">
+    <div class="video-container">
+      <iframe
+        class="video-iframe"
+        src="//www.youtube.com/embed/BOPDw348-zg"
+        title="TITLE"
+        allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share"
+        allowFullScreen></iframe>
+    </div>
+</div>
+
+:::info
+### What is MetaMask Portfolio?
```

Comment:
this callout feels a bit late, given that we first mention Portfolio right at the start of the mainnet section

Reply:
fixed